### PR TITLE
Added 3d printed case to documentation

### PR DESCRIPTION
**Pull request by bob9**
Updated Readme with link to the STL on [printables](https://www.printables.com/model/762529-esp32-wroom-32u-casing). 

Added example image of printed STL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,4 +210,14 @@ Row to show lap or gap time
...
+
+
+### 3D Printed Case
+Available on Printables
+https://www.printables.com/model/762529-esp32-wroom-32u-casing
+
+
+![Case](docs/3DPrint/esp_case.jpg)
+
+
```
Comment (klaasnicolaas) [inline README.md]:
```suggestion
### 3D Printed Case

Available on Printables
https://www.printables.com/model/762529-esp32-wroom-32u-casing

![Case](docs/3DPrint/esp_case.jpg)

```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Lets remove the extra black lines and follow the style guide for markdown. Would also place this much higher, perhaps below requirements.

./Klaas

**Comment by bob9** [comment]
I have updated the readme and removed black lines and changed the header to
be consistent with other headers. There was a guy who commented on the
discord that the stl from printable would be good if it was on the
documentation so thought i'd add it.

Cheers,

bob9

**Comment by klaasnicolaas** [comment]
> There was a guy who commented on the discord

That's me 😅

**Comment by bob9** [comment]
Your welcome. Readme updated. :)

**Pull request merged**